homeassistant/components/ambient_station/__init__.py

```diff
@@ -42,6 +44,20 @@
 CONFIG_SCHEMA = cv.deprecated(DOMAIN)
 
 
+def async_lx_to_perceived(value: int) -> int:
+    """Calculate illuminance (in lux)."""
+    lux = async_wm2_to_lx(value)
```

Comment:
The parameter should already be lux according to the function name and how we use this function.